playbook/CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 
 
+## [11.4.0](https://github.com/powerhome/playbook/tree/11.4.0) (2022-08-29)
```

Comment:
I just pushed up changes to master, so it appears you might need to merge master onto your branch.